Fix compatibility with Boost 1.63.

Patch copied from upstream source repository:

https://github.com/arvidn/libtorrent/commit/6d2d736cecce0af274dd651dd1f562716b625d92

From 6d2d736cecce0af274dd651dd1f562716b625d92 Mon Sep 17 00:00:00 2001
From: arvidn <arvid@cs.umu.se>
Date: Sun, 12 Mar 2017 13:03:26 -0400
Subject: [PATCH] fix test_ssl.cpp build with newer versions of boost

---
 test/test_ssl.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/test_ssl.cpp b/test/test_ssl.cpp
index 01c5bc7..2903332 100644
--- a/test/test_ssl.cpp
+++ b/test/test_ssl.cpp
@@ -51,7 +51,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/asio/connect.hpp>
 
 #ifdef TORRENT_USE_OPENSSL
-#include <boost/asio/ssl/error.hpp> // for asio::error::get_ssl_category()
 #include <boost/asio/ssl.hpp>
 
 #include "libtorrent/aux_/disable_warnings_pop.hpp"
